src/components/Contacts/Contacts.scss

```diff
@@ -0,0 +1,19 @@
+.contacts {
```

Comment:
we didn't needed contact form, it's a figma mistake

Reply:
let's keep Contact naming for now, we can delete it later if no need